crmsh/bootstrap.py

```diff
@@ -911,27 +921,19 @@ def pick_default_value(vlist, ilist):
     mcastport_res = []
     default_ports = ["5405", "5407"]
     two_rings = False
-    default_networks = []
 
-    if _context.ipv6:
-        network_list = []
-        all_ = utils.network_v6_all()
-        for item in all_.values():
-            network_list.extend(item)
-        default_networks = [utils.get_ipv6_network(x) for x in network_list]
-    else:
-        network_list = utils.network_all()
-        if len(network_list) > 1:
-            default_networks = [_context.ip_network, network_list.remove(_context.ip_network)]
-        else:
-            default_networks = [_context.ip_network]
-    if not default_networks:
+    local_iplist = utils.ip_in_local(_context.ipv6)
+    len_iplist = len(local_iplist)
+    if len_iplist == 0:
         error("No network configured at {}!".format(utils.this_node()))
 
+    local_iplist.remove(_context.ip_address)
+    default_ip = [_context.ip_address, "" if not local_iplist else local_iplist[0]]
```

Comment:
How about

```
default_ip = [_context.ip_address] + [ip for ip in local_iplist if ip != _context.ip_address]
```

Modifying local_iplist is confusing to me, there is a risk that it would be used later but now it does not contain `_context.ip_address`.

Same thing for line 1012 below.